src/sheldon_basic_check.erl

```diff
@@ -0,0 +1,97 @@
+%%% Copyright 2017 Inaka &lt;hello@inaka.net&gt;
+%%%
+%%% Licensed under the Apache License, Version 2.0 (the "License");
+%%% you may not use this file except in compliance with the License.
+%%% You may obtain a copy of the License at
+%%%
+%%%     http://www.apache.org/licenses/LICENSE-2.0
+%%%
+%%% Unless required by applicable law or agreed to in writing, software
+%%% distributed under the License is distributed on an "AS IS" BASIS,
+%%% WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+%%% See the License for the specific language governing permissions and
+%%% limitations under the License.
+%%% @end
+%%% @copyright Inaka <hello@inaka.net>
+%%%
+-module(sheldon_basic_check).
+-author("Felipe Ripoll <ferigis@inakanetworks.com>").
+
+-export([ prefixes/0
+        , suffixes/0
+        , escaped_words/0
+        ]).
+
+%%%===================================================================
+%%% API
+%%%===================================================================
+
+-spec prefixes() -> [binary()].
+prefixes() ->
+  [ <<"-">>
+  , <<"(">>
+  , <<"[">>
+  , <<"\"">>
+  , <<"\'">>
+  , <<"\t">>
+  , <<"{">>
+  , <<"`">>
+  , <<"$">>
```

Comment:
The code in the following (LINE, COL) locations has the same structure: (39, 5), (62, 5).